easy-ui-react/src/Tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,74 @@
+@use "../styles/common" as *;
+
+.Tooltip {
+  @include component-token("tooltip", "arrow-size", "1rem");
+  @include component-token("tooltip", "max-width", 320px);
```

Comment:
**question** is this `320px` spec'd somewhere?

Reply:
good catch. it's not. i wanted to guard against tooltip overflow on mobile screens, but i figured out a better way to handle that. this is now removed